### DISCUSSION TITLE
How do you enter value modifications of an account without any transaction ?

### DISCUSSION THREAD

**Discussion by mora-phi** [firefly-iii/firefly-iii#8950] [Firefly III]
Hi,
I have created an account that represents an inversion in stock exchange.
I set the opening balance to a point in time, and now I would like to enter the estimated value of the account manually, like every month.
Adding transactions seem wrong as in reality there is no such thing: value can go up or down with the market, but it's only something estimated and there is no transaction occurring.
I did not find a way to represent that in Firefly, is it even possible ? Maybe the tool is not at all thought for that purpose...

Thanks.

**Comment by JC5** [comment]
For this, I would suggest to use the open source application "portfolio performance".

**Comment by digitlength** [comment]
> Maybe the tool is not at all thought for that purpose...

Correct. Perhaps try [Ghostfolio](https://github.com/ghostfolio/ghostfolio/) for investment tracking.

**Comment by mora-phi** [comment]
Thanks a lot for the quick answers.
I'll have a look at the two applications you mentioned. And at least I'll stop searching for workarounds in Firefly III :-D
Cheers
